src/components/axes/timeAxis.ts

```diff
@@ -503,7 +503,7 @@ export module Axes {
     }
 
     private _hideOverlappingAndCutOffLabels(index: number) {
-      var boundingBox = this._element.select(".bounding-box")[0][0].getBoundingClientRect();
+      var boundingBox = (<Element> this._element.select(".bounding-box")[0][0]).getBoundingClientRect();
```

Comment:
Just a thought: we should consider if we want to enforce a whitespace between casting and variable or not. We have a mixture in the codebase